krummstab/__init__.py

```diff
@@ -1540,31 +1600,29 @@ def init(_the_config: config.Config) -> None:
     # .       └── submission.pdf or submission.zip
     rename_team_dirs(adam_id_to_team)
 
+    create_submission_info_files(adam_id_to_team, _the_config)
```

Comment:
move this after `flatten_team_dirs`

Reply:
alternatively move up and do this as part of `get_adam_id_to_team_dict` (in this case, this function should be renamed and no longer return a dict from team IDs to teams but instead write the team and the ID into the submission directory (next to the submission pdf).